mjpc/tasks/cube.xml.patch--- common_assets/reorientation_cube.xml	2024-02-11 18:42:07
+++ hand/cube.xml	2024-02-12 14:52:27
@@ -19,9 +19,9 @@
   </asset>
   <worldbody>
     <light pos="0 0 1"/>
-    <body name="cube" pos="0.2 0.0 0.075" quat="1 0 0 0">
+    <body name="cube" pos="0.325 0.0 0.075" quat="0.707 0.707 0 0">
       <freejoint/>
-      <geom name="cube" type="box" size=".03 .03 .03" mass=".122" material="cube"/>
+      <geom name="cube" type="box" size=".022 .022 .022" mass=".126" material="cube"/>
     </body>
   </worldbody>
 
